### DISCUSSION TITLE
assertion failed build could not compile in 10 seconds

### DISCUSSION THREAD

**Discussion by VovaGul** [nuke-build/nuke#1215] [Q&A] [answered]
**Nuke.Common** 
version 7.0.2

I'm using the [Nuke extension for Visual Studio](https://marketplace.visualstudio.com/items?itemName=nuke.visualstudio&ssr=false#overview)

my solution has 13 projects and doesn't have time to assemble in 10 seconds. Extension throws error "assertion failed build could not compile in 10 seconds".

Is there any way to increase the timeout from 10 seconds to 30?

**Comment by matkoch** [comment, marked as answer]
Feel free to send a PR that tries to resolve from an environment variable:

https://github.com/nuke-build/nuke/blob/develop/source/Nuke.Build/Execution/Extensions/HandleVisualStudioDebuggingAttribute.cs
